app/src/main/java/com/sopt/androidstudy/presentation/save/screens/FriendGithubActivity.kt

```diff
@@ -71,4 +121,168 @@ class FriendGithubActivity : AppCompatActivity() {
             binding.btnRepo.isEnabled = !isEnabled
         }
     }
-}
\ No newline at end of file
+
+    inner class MyHandler : Handler(Looper.getMainLooper()) {
+        override fun handleMessage(msg: Message) {
+            val bundle = msg.data
+            if (msg.what == COUNT_WHAT) {
+                val count = bundle.getInt("count")
+                binding.mbti.text = count.toString()
+            } else if (msg.what == IMAGE_WHAT) {
+                val stringBitmap = bundle.getString("bitmap")
+                stringBitmap?.let {
+                    binding.imageView.post {
+                        binding.imageView.setImageBitmap(convertBitmap.stringToBitmap(it))
+                    }
+                }
+            }
```

Comment:
```suggestion
            when (msg.what) {
                COUNT_WHAT -> {
                    val count = bundle.getInt("count")
                    binding.count.text = count.toString()
                }
                IMAGE_WHAT -> {
                    val stringBitmap = bundle.getString("bitmap") ?: return
                    binding.profileImage.post {
                        binding.profileImage.setImageBitmap(ConvertBitmap().stringToBitmap(stringBitmap))
                    }
                }
            }
```

이 부분도 그냥 스킵하셔도 좋습니다! 
bundle로 bitmap을 받아오지 못할 경우 `val stringBitmap = bundle.getString("bitmap") ?: return`로 함수를 바로 종료 시키도록 수정한다면 `stringBitmap?.let{}`으로 null 체크가 필요 없을 듯합니다! 이렇게 되면 들여쓰기를 줄일 수 있게됨에 따라 가독성도 좋아지는 것 같아요~!

Reply:
감사합니다!!